src/www/interfaces.php

```diff
@@ -1053,6 +1054,9 @@ function get_wireless_channel_info($interface) {
                     $new_config['subnetv6'] = $pconfig['subnetv6'];
                     if ($pconfig['gatewayv6'] != "none") {
                         $new_config['gatewayv6'] = $pconfig['gatewayv6'];
+                    }
+					if (!empty($pconfig['ipv6usev4iface'])) {
```

Comment:
looks like tabs here, we try use 4 spaces as per PSR2

Reply:
Noted.  I'll attempt to find the correct setting in my editor!